### PR TITLE
Finishing touches

### PR DESCRIPTION
**Pull request by tedks**
* Adds a README.md for the root of the repo.
* Adds the MVR ToS
* Adds the MVR FAQ
* Deletes privacy policy link for now.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+With MVR, builders can feel secure about which packages they’re actually calling, because their code has human-readable names and not inscrutable series of letters and numbers. MVR is open-source and uses immutable records on Sui as the source of truth. Further, MVR encourages Sui builders to make their packages open-source and easy to integrate into other projects.
+
+In the future, MVR will add features to surface other trust signals to make package selection easier, including auditor reports. Our hope is that MVR becomes the repository of Move packages in the same sense as NPM or Cargo, but built for the decentralized world.
+
+# Using MVR
+
+Right now there are two main surfaces to use MVR.
+
+* You can use MVR while *building* Move code, to manage your dependencies. For more on this, see mvr-cli/README.md.
+* You can use MVR while *calling* Move code in [Programmable Transaction Blocks](https://docs.sui.io/concepts/transactions/prog-txn-blocks). For more on this, see the [Typescript SDK Move Registry plugin](https://github.com/MystenLabs/sui/blob/main/sdk/typescript/src/transactions/plugins/NamedPackagesPlugin.ts).
```
Comment (manolisliolios) [inline README.md:18]: I think we could link to our docs instead? for both?

**Review by manolisliolios — APPROVED** [review]
I think it's fine to land and tune links once the doc site is also live!

**Review by ronny-mysten — COMMENTED** [review]
Just some nits

**Comment by ronny-mysten** [comment]
> I think it's fine to land and tune links once the doc site is also live!

Agree. None of my suggestions move the needle much.

File: app/src/data/content.ts [hunk 2]
```diff
@@ -94,52 +94,49 @@ export const FAQContent = {
...
     },
     {
-      title: "What is Lorem Ipsum?",
-      content: `Lorem Ipsum is simply dummy text of the printing and typesetting industry. 
-        Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. 
-        It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. 
-        It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software 
-        like Aldus PageMaker including versions of Lorem Ipsum`,
+      title: "What metadata is saved under an application?",
+      content: `An MVR record contains two things. The package address on chain, and the source code for Move packages depending on the MVR record's package. This is an off-chain URL (for now) which must, at the time of registration, contain an address field in the Move.toml or Move.lock file`,
```
[the remark this answers is not in the document]
Comment (stefan-mysten) [reply app/src/data/content.ts:119]: It is possible if people go change things manually.

File: README.md (hunk 1 above)
Comment (stefan-mysten) [reply README.md:18]: When we have the docs up, yes let's do that!

**Review by stefan-mysten — APPROVED** [review]

**Pull request merged**